moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect } from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  FormControl,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+} from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import ProjectSaveButton from "../newProjectView/ProjectSaveButton";
+import ToolbarPlugin from "./ToolbarPlugin";
+
+import { AutoFocusPlugin } from '@lexical/react/LexicalAutoFocusPlugin';
+import { LexicalComposer } from '@lexical/react/LexicalComposer';
+import { RichTextPlugin } from '@lexical/react/LexicalRichTextPlugin';
+import { ContentEditable } from '@lexical/react/LexicalContentEditable';
+import { HistoryPlugin } from '@lexical/react/LexicalHistoryPlugin';
+import { LexicalErrorBoundary } from '@lexical/react/LexicalErrorBoundary';
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { $generateHtmlFromNodes, $generateNodesFromDOM } from '@lexical/html';
+import { $getRoot } from 'lexical';
+import { $isRootTextContentEmpty } from '@lexical/text';
+import { LinkPlugin } from '@lexical/react/LexicalLinkPlugin';
+import { LinkNode } from '@lexical/link';
+import { ListPlugin } from '@lexical/react/LexicalListPlugin';
+import { ListNode, ListItemNode } from '@lexical/list';
+import PlaygroundEditorTheme from "./PlaygroundEditorTheme";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    backgroundColor: theme.palette.background.paper,
+  },
+  cancelButton: {
+    margin: theme.spacing(1),
+    position: "relative",
+    color: theme.palette.secondary.dark,
+  },
+  contentEditable: {
+    width: "100%",
+    padding: "16px 16px",
+    border: "4px solid",
+    borderRadius: "8px",
+    fontSize: "1rem",
+    borderColor: theme.palette.primary.main,
+    fontFamily: theme.typography.fontFamily,
+    color: theme.palette.text.primary,
+  },
+  placeholder: {
+    position: "absolute",
+    top: "36px",
+    paddingLeft: "20px"
+  },
+  editorWrapper: {
+    position: "relative"
+  },
+  showButtonItem: {
+    margin: theme.spacing(2),
+  },
+  toolbarButtons: {
+    backgroundColor: theme.palette.primary.main,
+    '&:hover, &.Mui-focusVisible': {
+      backgroundColor: theme.palette.primary.main,
+    },
+  },
+  startIcon: {
+    margin: 0,
+    color: theme.palette.text.primary,
```

Comment:
I noticed that this color style overrides the active/inactive colors built into the Toolbar theme. You can check that out by removing and seeing that the "redo" button icon is a disabled color until text is entered. I think it would be fine to remove this line, and let the toolbar theme take over.